docs/source/changelog.rst

```diff
@@ -1,26 +1,76 @@
 What's new
 ==========
 
-Unreleased
-----------
+poliastro 0.7.0 - 2017-09-07
```

Comment:
Probably will add some beta mark or something until we have the final release. Today it will be a beta.